robot_smach_states/src/robot_smach_states/manipulation/grab.py

```diff
@@ -36,6 +39,9 @@ def __init__(self, robot, arm, grab_entity):
         self.grab_entity_designator = grab_entity
 
         check_type(grab_entity, Entity)
+        assert self.robot.get_arm(
+            **self.REQUIRED_ARM_PROPERTIES), "None of the available arms meets all this class's" \
```

Comment:
Strictly speaking, I would even prefer `assert self.robot... is not None, ...`, to make it explicit that you're checking for a `None` return value rather than a `True` or `False`